anagrafica/models.py

```diff
@@ -1864,7 +1866,9 @@ def richiedi(self):
         self.autorizzazione_richiedi_sede_riferimento(
             self.persona,
             INCARICO_GESTIONE_TRASFERIMENTI,
-            invia_notifica_presidente=True
+            invia_notifica_presidente=True,
+            auto=Autorizzazione.AP_AUTO,
+            scadenza=settings.AUTORIZZAZIONE_AUTOMATICA,
```

Comment:
Ah, capito, no. E' una scelta principalmente estetica a dire il vero, e la motivazione e' la seguente: Il regolamento indica esplicitamente il limite di 30gg alla sezione dei trasferimenti, mi piacerebbe che Gaia lo rispecchi, avendo hard-coded tale limite in 'Trasferimento'. 

Comprendo che questo vuol dire che il default attualmente non verra' utilizzato da niente -- mi sta bene. 

Poi potra' essere utilizzato dalle richieste di partecipazione, esattamente come hai fatto l'esempio qui per il trasferimento, perche' quelle non fanno parte del regolamento ma sono piuttosto una nostra preferenza, e le chance sono che lo cambieremo nel tempo dopo aver fatto un'analisi statistica dei tempi che ci mette la gente ad approvare le richieste di partecipazione alle attivita', in genere.